src/nft/css/common.css.ts

```diff
@@ -41,7 +41,6 @@ export const commonButtonStyles = style([
   sprinkles({
     borderRadius: '12',
     transition: '250',
-    boxShadow: { hover: 'elevation' },
```

Comment:
common button styles has a boxShadow.  Not a heavily used style, but we removed boxShadows from buttons